admin/view/wp-slimstat-reports.php

```
@@ -1628,6 +1687,213 @@ public static function show_events($_args = [])
+    /**
+     * Renders the Goals report.
+     *
+     * - When $_args['is_widget'] is true (shortcode, dashboard widget, email
+     *   report, CSV fallback), renders the legacy compact table untouched so
+     *   downstream surfaces keep working.
+     * - Otherwise renders the modern admin card via the goals-card partial.
+     */
+    public static function show_goals($_args = [])
+    {
+        $goals      = get_option('slimstat_goals', []);
+        $max_goals  = (int) apply_filters('slimstat_max_goals', 1);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$max_goals'. (undefined)

(UnusedLocalVariable)